.github/workflows/checkstyle.yaml

```diff
@@ -63,10 +63,10 @@ jobs:
           cat go-deps-diff.txt
 
       - name: gazelle
-        # Keep Gazelle version in sync with WORKSPACE
```

Comment:
We should also remove the comment in [`WORKSPACE`](https://github.com/buildbuddy-io/buildbuddy/blob/master/WORKSPACE#L35) if we're removing this one.